### ISSUE TITLE
Allow the urlbar to be moved freely again.

### ISSUE THREAD

**Issue by RobertZenz** [MrAlex94/Waterfox#2909]
Would be quite neat if we would be able to move the urlbar freely again, currently it is locked to the navbar. I always liked to merge that toolbar into the menubar, but for that I need to be able to move the urlbar.

Ideally this would be a togglable option in the new themes preferences, I guess.

My current solution is to use "userChrome.css" to make `#urlbar` absolute positioned in the layout, which is quite a hassle to setup and also to maintain if you want to change, for example, the toolbar layout.

**Comment by jipouille** [comment]
hello maybe you should try classic theme restorer and you will be able to move urlbar with the menu bar maybe you would like something like this screenshot 
![waterfox](https://user-images.githubusercontent.com/121253774/209201906-3c372bed-fbdb-40fe-abc1-4ca83753281c.jpg)

**Comment by RobertZenz** [comment]
> hello maybe you should try classic theme restorer and you will be able to move urlbar with the menu bar maybe you would like something like this screenshot ![waterfox](https://user-images.githubusercontent.com/121253774/209201906-3c372bed-fbdb-40fe-abc1-4ca83753281c.jpg)

Yes, that would only work on Firefox Classic, if I'm not mistaken. I'm running Waterfox G5.
